File: Lab6_RIP/lab6/lab6/urls.py
from django.contrib import admin
from stocks import views as stock_views
from doge import views as doge_views
from django.urls import include, path
from rest_framework import routers

router = routers.DefaultRouter()
router.register(r'stocks', stock_views.StockViewSet)
router.register(r'doge', doge_views.dogeViewSet)
# Wire up our API using automatic URL routing.
# Additionally, we include login URLs for the browsable API.
urlpatterns = [
    path('', include(router.urls)),
    path('api-auth/', include('rest_framework.urls', namespace='rest_framework')),

    path('admin/', admin.site.urls),
]

File: Lab7_RIP/lab7/src/modules/urls.js
class Urls {
    constructor() {
        this.url = 'http://127.0.0.1:8000/';
    }

    stocks() {
        return `${this.url}doge/`
    }

    stock(id) {
        return `${this.url}doge/${id}/`
    }
}

export const urls = new Urls()